server/conf/application.conf

```diff
@@ -624,4 +624,11 @@ session_inactivity_warning_threshold_minutes = 5  # Warn the user 5 mins before
 session_inactivity_warning_threshold_minutes = ${?SESSION_INACTIVITY_WARNING_THRESHOLD_MINUTES}
 
 session_duration_warning_threshold_minutes = 10 # Warn the user 10 mins before the max session duration timeout
-session_duration_warning_threshold_minutes = ${?SESSION_DURATION_WARNING_THRESHOLD_MINUTES}
\ No newline at end of file
+session_duration_warning_threshold_minutes = ${?SESSION_DURATION_WARNING_THRESHOLD_MINUTES}
+
+# Theme settings
+theme_color_primary = ""
+theme_color_primary = ${?THEME_COLOR_PRIMARY}
+
+theme_color_primary_dark = ""
+theme_color_primary_dark = ${?THEME_COLOR_PRIMARY_DARK}
```

Comment:
(blocking) Please move these into the `server/conf/helper/whitelabel.conf` file. That file gets imported automatically so you shouldn't need to do anything else.